addons/project/controllers/portal.py

```diff
@@ -200,7 +200,7 @@ def portal_my_project_subtasks(self, project_id, task_id, page=1, date_begin=Non
         try:
             project_sudo = self._document_check_access('project.project', project_id)
             task_sudo = request.env['project.task'].search([('project_id', '=', project_id), ('id', '=', task_id)]).sudo()
-            task_domain = [('id', 'child_of', task_id), ('id', '!=', task_id)]
+            task_domain = [('parent_id', '=', task_id)]
```

Comment:
Normally, we should see all subtasks in portal but indeed in project sharing we should see the direct subtasks. 🙂 

Can you revert that changes? 🙂 

Thanks in advance! 🙂